src/nodedef.cpp

```diff
@@ -797,6 +800,12 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 				tdef[j].backface_culling, material_type);
 	}
 
+	if (drawtype == NDT_PLANTLIKE_ROOTED)
```

Comment:
GCC warning:
```
/data/Minetest/minetest/src/nodedef.cpp: In member function ‘void ContentFeatures::updateTextures(ITextureSource*, IShaderSource*, irr::scene::IMeshManipulator*, Client*, const TextureSettings&)’:
/data/Minetest/minetest/src/nodedef.cpp:804:5: warning: suggest explicit braces to avoid ambiguous ‘else’ [-Wparentheses]
  if (drawtype == NDT_PLANTLIKE_ROOTED)
```
This code works fine, as wanted but might be better to put brackets around to clarify.